src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -57,14 +79,41 @@
 (defn game-over? [board]
   (if (or (three-aligned? board) (board-full? board)) true false))
 
-(defn next-player-turn [board user-symbol]
+(defprotocol Player
+  (get-symbol [this])
+  (get-move [this board]))
+
+(deftype HumanPlayer [symbol]
+  Player
+  (get-symbol [this] symbol)
+  (get-move [this board]
+    (get-human-position)))
+
+(deftype ComputerPlayer [symbol]
+  Player
+  (get-symbol [this] symbol)
+  (get-move [this board]
+    (get-first-available-position board)))
+
+(defn next-player-turn [board current-player other-player]
   (display-board board)
   (if (game-over? board)
```

Comment:
You could add some logic that checks whether it is a tie or if there is a winner, and if there is a winner printing a message declaring it to the user. Maybe on the next iteration?

Reply:
Same here, it's also in the backlog but hasn't been deemed a priority, so it's one for next time!